Add purchase order GraphQL content (#77)
AdobeDocs/commerce-webapi: src/pages/graphql/schema/b2b/purchase-order-rule/index.md

      description
      status
      created_at
      updated_at
      created_by
      applies_to_roles {
        name
      }
      approver_roles {
        name
      }
      condition {
        attribute
        operator
        ... on PurchaseOrderApprovalRuleConditionAmount {
          amount {
            value
            currency
          }
        }
        ... on PurchaseOrderApprovalRuleConditionQuantity {
          quantity
        }
      }
    }
  }
}
```

**Response:**

``` json
{
  "data": {
    "customer": {
      "purchase_order_approval_rule": {
        "uid": "Ng==",
        "name": "Over 10 SKUs",
        "description": null,
        "status": "ENABLED",
        "created_at": "2023-01-25 20:05:25",
        "updated_at": "2023-01-25 20:05:25",
        "created_by": "Melanie Shaw",
        "applies_to_roles": [
          {
            "name": "Default User"
          }
        ],
        "approver_roles": [
          {
            "name": "Manager"
          }
        ],
        "condition": {
          "attribute": "NUMBER_OF_SKUS",
          "operator": "MORE_THAN_OR_EQUAL_TO",
          "quantity": 10
        }
      }
    }
  }
}
```

## Approval flow

> Company users that are enabled to create or review purchase orders designated as an approver can view purchase orders in the **My Purchase Orders** dashboard page.
> 

Reviewer: ```suggestion
Company users that are designated as approvers and can create or review purchase orders can use the **My Purchase Orders** dashboard page to view purchase orders.
```

**Request:**

``` graphql
{
  customer {
    purchase_orders(filter: { company_purchase_orders: true }) {
      items {
        approval_flow {
          rule_name
          events {
            name
            role
            status
            message
          }
        }
      }
    }
  }
}
```

**Response:**

``` json
{
  "data": {
    "customer": {
      "purchase_orders": {
        "items": [
          {
            "approval_flow": [
              {
                "rule_name": "Rule 1",
                "events": [
                  {
                    "name": "Unknown Customer",
                    "role": "Company Administrator",
                    "status": "PENDING",
                    "message": "Pending Approval from Company Administrator"
                  },
                  {
                    "name": "Unknown Customer",
                    "role": "Purchaser's Manager",
                    "status": "PENDING",
                    "message": "Pending Approval from Purchaser's Manager"
                  }
                ]
              }
            ]
          }
        ]
      }
    }
  }
}
```

## Return approval rule metadata

> The following example returns the metadata necessary to render the form for creating or editing an approval rule.

Reviewer: ```suggestion
The following example returns the metadata necessary to render the form for [creating or editing an approval rule](https://experienceleague.adobe.com/docs/commerce-admin/b2b/purchase-orders/account-dashboard-approval-rules.html#approval-rule-setup).
```
